docs/source/code-generation/codegen-cli.mdx

```diff
@@ -107,7 +107,13 @@ This will compile the Codegen CLI in release mode for you.
 
 The Codegen CLI executable will now be located in your `apollo-ios` clone at `.build/release/apollo-ios-cli`.
 
-You can run the CLI from there, move it to another location, or add it to your shell's `$PATH`.
+Before running code gen we recommend you move the `apollo-ios-cli` executable to the root directory. or add it to your shell's `$PATH`.
```

Comment:
What's the reason we recommend moving it? I think we should start distributing the pre-built binary CLI with each release, which essentially negates the step of cloning and building the CLI; it can simply be downloaded.

Reply:
I found it easier to work with when testing in a separate directory as the .build is a hidden directory. Though the  getting started guide walks the user through how to use the CLI, so I think it's not needed here to your point. 

I think it be great to have a pre-built binary of the CLI available in each release, so whenever we introduce that we can update the docs as needed.